radicle-httpd/src/lib.rs

```diff
@@ -98,9 +98,9 @@ async fn git_handler(
     AxumPath((project, request)): AxumPath<(String, String)>,
     method: Method,
     headers: HeaderMap,
-    body: Bytes,
     ConnectInfo(remote): ConnectInfo<SocketAddr>,
     query: RawQuery,
+    body: Bytes,
```

Comment:
The extractor which consumes the body should be last in new axum, should work just fine, but maybe we should add a simple test here, just to be sure?